### PR TITLE
fix(test) : replace browser.sleep with wait until

### PR DESCRIPTION
**Pull request by Raunak1203**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [x] What does this PR do? 
This PR replaces browser.sleep with wait until condition
- [ ] What issue/task does this PR references?
https://github.com/fabric8-ui/fabric8-planner/issues/2632
- [x] Are the tests Included?
___
### Optional
- [ ] Is the documentation Included?

- [ ] Are the Release Notes included?

- [ ] @mention(s) to expected reviewer(s) included

### CODE REVIEW

**Comment by Raunak1203** [comment]
[test]

File: src/tests/page_objects/planner/planner.page.ts [hunk 1]
```diff
@@ -49,4 +49,8 @@ export class PlannerPage extends AppPage {
     await $('body').sendKeys(protractor.Key.ESCAPE);
     await this.quickPreview.notificationToast.untilHidden();
   }
+
+  async waitUntilUrlContains(text: string) {
```
Comment (jarifibrahim) [inline src/tests/page_objects/planner/planner.page.ts]: Can we move this to `basePage.ts` ?
This is a generic function and should be in base page class.

Comment (Raunak1203) [reply src/tests/page_objects/planner/planner.page.ts]: done @jarifibrahim

**Comment by Raunak1203** [comment]
[test]

File: src/tests/page_objects/planner/planner.page.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { $, browser } from 'protractor';
+import { $, browser, ExpectedConditions } from 'protractor';
```
Comment (jarifibrahim) [inline src/tests/page_objects/planner/planner.page.ts]: You should remove this import

Comment (Raunak1203) [reply src/tests/page_objects/planner/planner.page.ts]: done @jarifibrahim

**Comment by jarifibrahim** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
done @vikram-raj

**Pull request merged**